index.bs

```diff
@@ -1627,6 +1664,41 @@ Releases the resources associated with the {{MLTensor}}. This method is idempote
 
 Note: Since no further operations can be enqueued using this tensor, implementations can free any additional resource allocations associated with this tensor once all previously submitted operations using it are complete.
 
+## {{MLConstantTensor}} interface ## {#api-mlconstanttensor}
+
+The {{MLConstantTensor}} interface represents a tensor which may be used as a constant to an {{MLGraph}}. The memory backing an {{MLConstantTensor}} should be allocated in an [=implementation-defined=] fashion according to the requirements of the {{MLContext}} and the {{MLOperandDescriptor}} used to create it. Operations involving the {{MLTensor/[[data]]}} of an {{MLConstantTensor}} occur on the {{MLContext/[[timeline]]}} of its associated {{MLContext}}.
+
+The [=implementation-defined=] requirements of how an {{MLConstantTensor}} is allocated may include constraints such as that the memory is allocated with a particular byte alignment or in a particular memory pool.
+
+<script type=idl>
+[SecureContext, Exposed=(Window, Worker)]
+interface MLConstantTensor : MLTensor {
+};
+</script>
+
+
+The <dfn attribute for=MLConstantTensor>readable</dfn> [=getter steps=] are to return true.
```

Comment:
`MLConstantTensor` doesn't define `readable` or `writable` attributes, just inherits them, so we wouldn't normally spec it this way. You might be able to get away with defining overrides on `MLConstantTensor` but I'm not sure what would happen if script invokes the `MLTensor` getter functions with an `MLConstantTensor` "this" object. 

Since `MLTensor`'s definitions of `readable` and `writable` are to read the properties out of the **[[descriptor]]** internal slot's dict, as part of addressing setting that slot (see other comment) you can just set them to true/false during the tensor creation.

Reply:
Okay. Seems we can remove this as well then.